src/Packaging/Packaging.csproj

```diff
@@ -0,0 +1,165 @@
+    <!-- OutputPath is required by the Clean and Rebuild tasks -->
+    <OutputPath>bin\$(Configuration)\</OutputPath>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <SonarQubeExclude>True</SonarQubeExclude>
+  </PropertyGroup>
+
+  <PropertyGroup Condition=" '$(TargetFramework)' == '$(ScannerNetFxVersion)' ">
+    <CommonFrameworkVersion>$(TargetFramework)</CommonFrameworkVersion>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(TargetFramework)' != '$(ScannerNetFxVersion)' ">
+    <CommonFrameworkVersion>$(ScannerNetStandardVersion)</CommonFrameworkVersion>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <!-- This version needs to be in sync with version in src/SonarScanner.MSBuild.Shim/SonarScanner.Wrapper.cs -->
+    <ScannerCliVersion Condition=" $(ScannerCliVersion) == '' ">3.2.0.1227</ScannerCliVersion>
+    <ScannerCliArtifact>sonar-scanner-cli-$(ScannerCliVersion).zip</ScannerCliArtifact>
+    <MavenCentralUrl>https://repox.sonarsource.com/sonarsource-public-releases/org/sonarsource/scanner/cli/sonar-scanner-cli/$(ScannerCliVersion)/$(ScannerCliArtifact)</MavenCentralUrl>
+    <SourcesRoot>$(MSBuildThisFileDirectory)..</SourcesRoot>
+    <BuildAgentPayloadDir>$(SourcesRoot)\..\DeploymentArtifacts\BuildAgentPayload\</BuildAgentPayloadDir>
+    <WorkDestinationRootDir>$(BuildAgentPayloadDir)$(Configuration)\</WorkDestinationRootDir>
+    <WorkDestinationDir>$(WorkDestinationRootDir)\$(TargetFramework)</WorkDestinationDir>
+    <DestinationArtifactPath>$(WorkDestinationRootDir)\sonarscanner-msbuild-$(TargetFramework).zip</DestinationArtifactPath>
+  </PropertyGroup>
+
+  <!-- Common -->
+  <ItemGroup>
+    <!-- Project dependencies - ensure build order -->
+    <ProjectReference Include="$(SourcesRoot)\SonarScanner.MSBuild.Common\SonarScanner.MSBuild.Common.csproj" />
+    <ProjectReference Include="$(SourcesRoot)\SonarScanner.MSBuild\SonarScanner.MSBuild.csproj" />
+    <ProjectReference Include="$(SourcesRoot)\SonarScanner.MSBuild.Tasks\SonarScanner.MSBuild.Tasks.csproj" />
+    <ProjectReference Include="$(SourcesRoot)\SonarScanner.MSBuild.TFS\SonarScanner.MSBuild.TFS.csproj" />
+    <ProjectReference Include="$(SourcesRoot)\SonarScanner.MSBuild.PostProcessor\SonarScanner.MSBuild.PostProcessor.csproj" />
+    <ProjectReference Include="$(SourcesRoot)\SonarScanner.MSBuild.PreProcessor\SonarScanner.MSBuild.PreProcessor.csproj" />
+    <ProjectReference Include="$(SourcesRoot)\SonarScanner.MSBuild.Shim\SonarScanner.MSBuild.Shim.csproj" />
+
+    <!-- Targets -->
+    <TargetFiles Include="$(SourcesRoot)\SonarScanner.MSBuild.Tasks\Targets\SonarQube.Integration.targets" />
+    <TargetFiles Include="$(SourcesRoot)\SonarScanner.MSBuild.Tasks\Targets\SonarQube.Integration.ImportBefore.targets" />
+
+    <!-- Dlls -->
+    <ScannerFiles Include="$(SourcesRoot)\SonarScanner.MSBuild\bin\$(Configuration)\$(TargetFramework)\SonarQube.Analysis.xml" />
```

Comment:
Wouldn't copy local work as well? E.g. could we collect binaries from this project's output folder, instead of browsing the others?

Reply:
It would probably but I spent enough time improving that for the current sprint.